Makefile

```diff
@@ -62,25 +69,35 @@ EXAMPLES = classic epoch_shift hello example1 example2 example3 example4
 
 all: $(TESTS) $(TOOLS) $(EXAMPLES)
 
-$(TESTS) $(TOOLS) $(EXAMPLES): $(CCTZ_LIB)
+$(TESTS) $(TOOLS) $(EXAMPLES): $(CCTZ_SHARED_LIB)
 
-$(CCTZ_LIB): $(CCTZ_OBJS)
+$(CCTZ_SHARED_LIB): $(CCTZ_OBJS) $(CCTZ_SHARED_SONAME)
```

Comment:
There seems to be some circularity here ... CCTZ_SHARED_LIB depends on CCTZ_SHARED_SONAME and CCTZ_SHARED_SONAME depends on CCTZ_SHARED_LIB.